src/Microsoft.Health.Fhir.SqlServer/Registration/SqlServerRegistrationExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class SqlServerRegistrationExtensions
+    {
+        public static IServiceCollection AddSqlServer(this IServiceCollection services)
```

Comment:
It is a bit weird that we have `AddSqlServer()` and `AddExperimentalSqlServer()` as extension methods on `IServiceCollection` and both have the same namespace. I think this will lead to confusion.